src/Microsoft.Health.Fhir.SqlServer/Features/Storage/Registry/SearchParameterStatusCollection.cs

```diff
@@ -0,0 +1,32 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Data;
+using Microsoft.Data.SqlClient.Server;
+using Microsoft.Health.Fhir.Core.Features.Search.Registry;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Storage.Registry
+{
+    public class SearchParameterStatusCollection : List<ResourceSearchParameterStatus>, IEnumerable<SqlDataRecord>
+    {
+        IEnumerator<SqlDataRecord> IEnumerable<SqlDataRecord>.GetEnumerator()
+        {
+            var sqlRow = new SqlDataRecord(
+                new SqlMetaData("Uri", SqlDbType.NVarChar, 128),
```

Comment:
>NVarChar [](start = 49, length = 8)

Isn't that are VarChar? At least it's varchar in 6.sql.
Same for Status.

Reply:
Thanks, fixed in 12d56d7.